packages/frontend/src/redux/slices/tokenFiatValues/index.js

```diff
@@ -23,7 +23,7 @@ const fetchTokenFiatValues = createAsyncThunk(
     `${SLICE_NAME}/fetchTokenFiatValues`,
     async (_, {dispatch, getState}) => {
         return Promise.allSettled([
-            dispatch(fetchCoinGeckoFiatValues(['near', 'usn'])),
+            dispatch(fetchCoinGeckoFiatValues(['near', 'usn', 'jumbo'])),
```

Comment:
I think that the asset on coingecko that you are actually interested in called ['jumbo-exchange'. ](https://www.coingecko.com/en/coins/jumbo-exchange). The price should be closer to .87 cents according to Ref Finance Pool as well as the correct listing on Coingecko listing.

The app is displaying the incorrect pricing. There are two assets listed on coingecko, one is 'jumbo'. it was last updated on June 6th. 

<img width="736" alt="image" src="https://user-images.githubusercontent.com/25015977/186989338-26c09b0e-82bd-4566-8d0e-f86039738878.png">

Reply:
Thank you, fixed!